### ISSUE TITLE
Exclude docs path from GitHub Actions tests

### ISSUE THREAD

**Issue by menghif** [Seneca-CDOT/telescope#2741]
**What would you like to be added**:

We currently run all of our tests in CI when we make a small change to the docs. In that situation, we could avoid running all the tests and run only Prettier (and ESlint?) instead.

**Why would you like this to be added**:

It would speed up CI.

I can make this change but I would like to know if it makes sense.

**Comment by humphd** [comment]
I think this makes sense.  I assume we can do it as part of the turborepo stuff, right?  Or do we have to included path level `on` events for the GitHub Actions?

**Comment by menghif** [comment]
Yes Turborepo should do that with Remote Caching setup. Until then I'm going to make a quick change to GitHub Actions.
